examples/scripts/denoise_l1tv_iso_admm.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of the SCICO package. Details of the copyright
+# and user license can be found in the 'LICENSE.txt' file distributed
+# with the package.
+
+r"""
+ℓ1 Total Variation (ADMM)
+=========================
+
+This example demonstrates impulse noise removal via ℓ1 total variation
+:cite:`alliney-1992-digital` :cite:`esser-2010-primal` (Sec. 2.4.4)
+(i.e. total variation regularization with an ℓ1 data fidelity term),
+minimizing the functional
+
+  $$\mathrm{argmin}_{\mathbf{x}} \;  \| \mathbf{y} - \mathbf{x}
+  \|_1 + \lambda \| C \mathbf{x} \|_1 \;,$$
+
+where $\mathbf{y}$ is the noisy image, $C$ is a 2D finite fifference
+operator, and $\mathbf{x}$ is the denoised image.
+"""
+
+import jax
+
+from xdesign import SiemensStar, discrete_phantom
+
+import scico.numpy as snp
+from scico import functional, linop, loss, metric, plot
+from scico.examples import spnoise
+from scico.optimize.admm import ADMM, LinearSubproblemSolver
+from scico.util import device_info
+from scipy.ndimage import median_filter
+
+"""
+Create a ground truth image and impose salt & pepper noise to create a
+noisy test image.
+"""
+N = 256  # image size
+phantom = SiemensStar(16)
+x_gt = snp.pad(discrete_phantom(phantom, 240), 8)
+x_gt = jax.device_put(x_gt)  # convert to jax type, push to GPU
+x_gt = 0.5 * x_gt / x_gt.max()
```

Comment:
Seems cleaner if you first scale and then put it on the device

Reply:
Agreed. Done.